values.yaml

```diff
@@ -39,6 +39,31 @@ service:
     liveness:
       path: /config
 
+gateway:
```

Comment:
This is not going to work since the HttpRoute Kubernetes resource only allows up to 8 routes, and we have currently more than that. We should add another sidecar container following the Adapter Kubernetes Pattern.